packages/portfolio-contract/src/portfolio.flows.ts

```diff
@@ -124,284 +112,302 @@ const initNobleAccount = async (
-      subscriber: ctx.inertSubscriber,
-      storagePath: accountId,
-    };
-    return topic;
-  } catch (err) {
-    console.error('⚠️ EVM account creation failed', err);
-    await ctx.zoeTools.withdrawToSeat(localAccount, seat, give);
-    assert.fail(`EVM account creation failed ${err}`);
-  }
+  const topic: GuestInterface<ResolvedPublicTopic<unknown>> = {
+    description: `EVM Addr`,
+    subscriber: inertSubscriber,
+    storagePath: accountId,
+  };
+  return topic;
 };
 
 const sendTokensViaCCTP = async (
-  orch: Orchestrator,
-  ctx: Pick<PortfolioContext, 'zoeTools' | 'chainHubTools'>,
+  orch,
+  seat,
+  ctx: PortfolioInstanceContext,
+  args: {
+    amount: Amount<'nat'>;
+    destinationEVMChain: AxelarChain;
+  },
   keeper: GuestInterface<PortfolioKit>['keeper'],
-  seat: ZCFSeat,
-  offerArgs: OfferArgsFor['openPortfolio'],
-  destAddr: `0x${string}`,
-  amount: Amount<'nat'>,
 ) => {
-  const denom = ctx.chainHubTools.getDenom(amount.brand);
-  assert(denom, `need denom for ${amount.brand} in chainHub`);
-
-  const nobleAccount = keeper.getUSDNICA();
-
-  const localAccount = keeper.getLCA();
+  const { axelarChainsMap, chainHubTools, zoeTools } = ctx;
+  const { amount, destinationEVMChain } = args;
+  const nobleChain = await orch.getChain('noble');
+  const nobleAccount = await nobleChain.makeAccount();
 
+  const localAcct = await keeper.getLCA();
   const amounts = harden({ Aave: amount });
-  await ctx.zoeTools.localTransfer(seat, localAccount, amounts);
+  trace('localTransfer', amount, 'to local', localAcct.getAddress().value);
+  await zoeTools.localTransfer(seat, localAcct, amounts);
 
   try {
-    trace('IBC transfer to Noble for CCTP');
-    await localAccount.transfer(nobleAccount.getAddress(), amount);
+    await localAcct.transfer(nobleAccount.getAddress(), amount);
 
+    const denom = await chainHubTools.getDenom(amount.brand);
+    assert(denom, 'denom must be defined');
     const denomAmount = {
       denom,
       value: amount.value,
     };
 
-    const caipChainId: CaipChainId = PositionChain[offerArgs.evmChain];
-    const destinationAddress: AccountId = `${caipChainId}:${destAddr}`;
-    await nobleAccount.depositForBurn(destinationAddress, denomAmount);
+    const caipChainId = axelarChainsMap[destinationEVMChain].caip;
+    const remoteAccountAddress = await keeper.getRemoteAddress();
+    const destinationAddress = `${caipChainId}:${remoteAccountAddress}`;
+    trace(`destinationAddress: ${destinationAddress}`);
+    await nobleAccount.depositForBurn(
+      destinationAddress as `${string}:${string}:${string}`,
+      denomAmount,
+    );
   } catch (err) {
-    await ctx.zoeTools.withdrawToSeat(localAccount, seat, amounts);
-    const errorMsg = '⚠️ Noble transfer failed:';
-    console.error(errorMsg, err);
-    throw new Error(`${errorMsg} ${err}`);
+    await zoeTools.withdrawToSeat(localAcct, seat, amounts);
+    const errorMsg = `'Noble transfer failed: ${err}`;
```

Comment:
we should be using `X` from `@endo/errors` for this sort of error message. Add a TODO, please?

```js
X`...`
```